packages/foreman/nodejs-node-gyp/node-gyp-node-dirs.patch

```diff
@@ -1,46 +1,49 @@
 diff --git a/lib/configure.js b/lib/configure.js
-index 2ff476d..7a69e1c 100644
+index 564564e..d3974ba 100644
 --- a/lib/configure.js
 +++ b/lib/configure.js
-@@ -62,7 +62,9 @@ function configure (gyp, argv, callback) {
+@@ -37,7 +37,7 @@ function configure (gyp, argv, callback) {
+
+     if (gyp.opts.nodedir) {
+       // --nodedir was specified. use that for the dev files
+-      nodeDir = gyp.opts.nodedir.replace(/^~/, os.homedir())
++      nodeDir = '/root/usr/share/node'
+
+       log.verbose('get node dir', 'compiling against specified --nodedir dev files: %s', nodeDir)
+       createBuildDir()
+@@ -48,7 +48,10 @@ function configure (gyp, argv, callback) {
          log.verbose('get node dir', 'compiling against --target node version: %s', release.version)
        } else {
          // if no --target was specified then use the current host node version
 +        // if no --target was specified then use RPM installed headers
++
          log.verbose('get node dir', 'no --target version specified, falling back to host node version: %s', release.version)
 +        log.verbose('get node dir', 'no --target version specified, falling back to RPM installed headers')
        }
 
        if (!release.semver) {
-@@ -75,7 +77,7 @@ function configure (gyp, argv, callback) {
-       gyp.commands.install([ release.version ], function (err, version) {
-         if (err) return callback(err)
-         log.verbose('get node dir', 'target node version installed:', release.versionDir)
--        nodeDir = path.resolve(gyp.devDir, release.versionDir)
-+        nodeDir = '/opt/rh/rh-nodejs10/root/usr/share/node'
```

Comment:
Just `/usr/share/node` here

Reply:
This is so odd, because I don't see it locally.